src/Mvc/Mvc.ApiExplorer/src/EndpointModelMetadata.cs

```diff
@@ -51,4 +51,19 @@ public EndpointModelMetadata(ModelMetadataIdentity identity) : base(identity)
     public override string? TemplateHint { get; }
     public override bool ValidateChildren { get; }
     public override IReadOnlyList<object> ValidatorMetadata { get; } = Array.Empty<object>();
+
+    public static Type GetDisplayType(Type type)
+    {
```

Comment:
```suggestion
    {
        // Return typeof(string) for any TryParsable type that's not built into the framework.
```

Do we need to include more built-in types listed here?

https://github.com/dotnet/aspnetcore/blob/0f517603dc0cf996f58ccd67fd12b513794de5a4/src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs#L513-L546

Reply:
I have talked with @captainsafia and there is a small (or zero) benefit in add all built-in types comparing with the maintenance cost.